xwiki-platform-core/xwiki-platform-icon/xwiki-platform-icon-ui/src/main/resources/IconThemesCode/Translations.xml

```diff
@@ -45,9 +45,9 @@ platform.icon.picker.loading=Loading
 rendering.macro.iconPicker.name=Icon Picker
 rendering.macro.iconPicker.description=Select an icon within the XWiki icon set.
 rendering.macro.iconPicker.parameter.id.name=Id
-rendering.macro.iconPicker.parameter.id.description=DOM id of the input field where the picker will apply
+rendering.macro.iconPicker.parameter.id.description=DOM id of the input field where the icon would be set
```

Comment:
That's not really what it does. It's about the input where the icon picker widget will attach to, not about the location of an icon. Please revert.